.eslintrc.js

```diff
@@ -58,8 +58,6 @@ module.exports = {
     'eslint-comments/no-unused-disable': 'error',
     'import/prefer-default-export': 'off',
     'tsdoc/syntax': 'warn',
-    'require-await': 'off',
-    '@typescript-eslint/require-await': 'error',
     '@typescript-eslint/no-floating-promises': 'error',
```

Comment:
Thank you for your reply, but in this configuration file, configurations like this appear twice. If the duplicate definitions are not removed, it might cause the following issues: rule conflicts, performance problems, maintenance difficulties, increased complexity in understanding, and unpredictable behavior.

Reply:
And you can find the same configuration on lines 33 and 34 of the current file.

Reply:
I see what you mean. DIdn't notice that these 2 rules were duplicated